### ISSUE TITLE
Feature request: Flip the implementation

### ISSUE THREAD

**Issue by gerrywastaken** [JackCuthbert/kaomoji-bot#1]
When a command succeeds I want:

&lt;kaomoji&gt; -- @&lt;username&gt;

without the original command being shown as it's just clutter which is already expressed by the kaomoji.

However when the command fails, then I'd like to see the command shown as at least then my intent is expressed and it's similar to a public feature request. I don't want these requests hidden.

**Comment by JackCuthbert** [comment]
We're now looking at using a bot to send the messages so we've got finer control over how messages are formatted. At the moment I've got something like this:

1. `/kaomoji cat This is another message`
2. `/kaomoji sad`

![screenshot from 2017-02-12 20-20-56](https://cloud.githubusercontent.com/assets/5564612/22860965/eec44c70-f160-11e6-8a1e-2c4ca42857e7.png)

Still looking into failing commands so I'll keep this open until I know more (*＾▽＾)／"
